scope/tag.go

```diff
@@ -24,7 +24,8 @@ func Tag(my, sv *Extent) error {
 	if err != nil {
 		return err
 	}
-	if tag != nil {
+	log.Printf("# -> Force: %t", force)
```

Comment:
Do you need to print force everytime?  Maybe only print force state if the flag is true, otherwise it just adds extra noise in the logs.

Reply:
it will only print when tag option is specified in the CLI, the amount of logging that this app does is very minimal. in fact, one of the things I want to address is to make it more verbose as IMO it doesn't do enough.  the user has the ability to enable/disable logging via environment variable.